### PR TITLE
Updates: RELEASE-NOTES for in-app updates

### PR DESCRIPTION
**Pull request by pantstamp**
This PR just adds a line in RELEASE-NOTES.txt for the [in-app updates](https://jetpackmobile.wordpress.com/2024/05/08/pt-in-app-updates/) feature. 

Related PR [here](https://github.com/wordpress-mobile/WordPress-Android/pull/20822). 

-----

## To Test:

Just check the code changes.

-----

## Regression Notes

1. Potential unintended areas of impact

    - TODO

2. What I did to test those areas of impact (or what existing automated tests I relied on)

    - TODO

3. What automated tests I added (or what prevented me from doing so)

    - TODO

-----

## PR Submission Checklist:

- [ ] I have completed the Regression Notes.
- [ ] I have considered adding accessibility improvements for my changes.
- [ ] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

-----

## Testing Checklist (strike-out the not-applying and unnecessary ones):

- [ ] WordPress.com sites and self-hosted Jetpack sites.
- [ ] Portrait and landscape orientations.
- [ ] Light and dark modes.
- [ ] Fonts: Larger, smaller and bold text.
- [ ] High contrast.
- [ ] Talkback.
- [ ] Languages with large words or with letters/accents not frequently used in English.
- [ ] Right-to-left languages. (Even if translation isn’t complete, formatting should still respect the right-to-left layout)
- [ ] Large and small screen sizes. (Tablet and smaller phones)
- [ ] Multi-tasking: Split screen and Pop-up view. (Android 10 or higher)

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">PR requires at least one label.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -8,6 +8,7 @@
 * [*] Fixed a rare crash on the featured image confirmation dialog [https://github.com/wordpress-mobile/WordPress-Android/pull/20836]
 * [*] Fixed an ANR issue on the Post List screen [https://github.com/wordpress-mobile/WordPress-Android/pull/20833]
 * [*] [internal] Block Editor: Upgrade target sdk version to Android API 34 [https://github.com/wordpress-mobile/WordPress-Android/pull/20841]
+* [*] In-app updates feature [https://github.com/wordpress-mobile/WordPress-Android/pull/20822]
```
Comment (AjeshRPai) [inline RELEASE-NOTES.txt]: We should the [internal] tag at the starting of the line

`* [*] [internal] In-app updates feature [https://github.com/wordpress-mobile/WordPress-Android/pull/20822]`

Comment (AjeshRPai) [reply RELEASE-NOTES.txt]: I have done it in a commit. Hope its ok with you. Approving the PR. Leaving it for you to merge

**Review by AjeshRPai — APPROVED** [review]
👍🏼

**Pull request merged**